src/components/MerchantsPage/MerchantsPage.tsx

```diff
@@ -112,6 +113,12 @@ const MerchantsPage = (props: Props) => {
               </div>
             </div>
           </MerchantInfoContainer>
+
+          <FlyerContainer>
+            <p>
+              Know of any business owners that fit our target merchant? <a download href={sclFlyerEnglish}>Download our flyer to share with them.</a>
```

Comment:
These should be translated using t(...)

Reply:
Ah thanks for the catch, was wondering how the language toggle worked! Let me add that

Reply:
The language toggle doesn't currently work because we haven't input the Chinese translations yet, but we should stay on top of supporting it

Reply:
Will followup with the translation support and color refactoring in another MR 👍